gfx/Mushroom-Dream/pngs_giant_32x64/terrain/t_door/t_door_o.json

```diff
@@ -0,0 +1,5 @@
+  {
+    "id": "t_door_o",
```

Comment:
You don't need this json if it points to a png with the same name as the id

Reply:
Haha, i didn't know what to call them so that happened. I can't find the ids to those. But i'll try to name them something more meaningful. 

I'll also fix those jsons. There might be a few more like that because i had a bug before on the composer that would not add a tile to the tileset if there was no json

Reply:
Alright, i made those changes